### PR TITLE
DevKit updates for 3.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

**Review by jordisala1991 — APPROVED** [review]

File: src/Sitemap/SourceManager.php [hunk 1]
```diff
@@ -52,6 +52,12 @@ public function addSource(string $group, SourceIteratorInterface $source, array
         }
     }
 
+    /**
+     * @return mixed
+     *
+     * @phpstan-return Source
```
Comment (VincentLanglet) [inline src/Sitemap/SourceManager.php:58]: Why not just `return Source` ?

File: src/Sitemap/SourceManager.php [hunk 2]
```diff
@@ -62,6 +68,12 @@ public function next(): void
         $this->sources->next();
     }
 
+    /**
+     * @return mixed
+     *
+     * @phpstan-return string
```
Comment (VincentLanglet) [inline src/Sitemap/SourceManager.php:74]: why not just return string ?

**Review by VincentLanglet — CHANGES_REQUESTED** [review]

File: src/Sitemap/SourceManager.php (hunk 1 above)
Comment (jordisala1991) [reply src/Sitemap/SourceManager.php:58]: the parent class is returning mixed, on next major I guess we should add `: mixed`

File: src/Sitemap/SourceManager.php (hunk 2 above)
Comment (jordisala1991) [reply src/Sitemap/SourceManager.php:74]: Same as above, parent class is returning mixed. We should not forget to add the correct return type

File: src/Sitemap/SourceManager.php (hunk 1 above)
Comment (VincentLanglet) [reply src/Sitemap/SourceManager.php:58]: This seems to not be an issue, see https://github.com/sonata-project/SonataSeoBundle/pull/642

**Pull request merged**